app.js

```diff
@@ -86,26 +86,28 @@ app.post('/api/addUserRequest', function (req, res) {
         }
 
         var pendingRequest = req.body;
-        pendingRequest["time_submitted"] = new Date();
+        pendingRequest["display_name"] = bandname();
+        pendingRequest["time_submitted"] = new Date().getTime();
```

Comment:
It turns out that the epoch time (which was added in a previous pull request) got overwritten or merged wrong and was causing fatal errors. Changed it back.